components/edit-collective/sections/Tiers.js

```diff
@@ -440,12 +445,14 @@ class Tiers extends React.Component {
   render() {
     const { intl, collective, defaultType = TICKET } = this.props;
     const hasCustomContributionsDisabled = get(collective, 'settings.disableCustomContributions', false);
-    const displayCustomContributionsSettings = collective.id && defaultType !== TICKET;
+    const hasCryptoContributionsDisabled = get(collective, 'settings.disableCryptoContributions', true);
+    const cryptoContributionsEnabledByHost = !get(collective, 'host.settings.cryptoEnabled', true);
```

Comment:
Should not be negative right?

```suggestion
    const cryptoContributionsEnabledByHost = get(collective, 'host.settings.cryptoEnabled', true);
```

Reply:
Yes, that was my mistake. 🤯